### ISSUE TITLE
Ny app: Personnavn og deres egenskaper

### ISSUE THREAD

**Issue by Ingerid** [NationalLibraryOfNorway/DHLAB-apps#7]
- Inndata: Enkelttekst fra korpus
- Hente navn som er subjekter og subst/adj-predikat de står til  
- Klassifisere kjønn på personnavn 
- Utdata: tabell med [navn, kjønn, predikat/egenskap, frekvens]

Funksjonskrav:   
- Korpusopplaster 
- nedtrekksmeny for dokumenter 
- Filter på navnetype (person, sted, organisasjon, sammensetning med navn)
- Filter på egenskapstype (karakteristikk=predikat er adj,  eller handling=predikat er verb) 
- Filter på kjønn

**Comment by Yoonsen** [comment]
Ja, det er i ferd med å komme en slik funksjonalitet på Gitlab-siden. For kjønn kan det legges utenom Gitlab, altså at resultatet fra api-kallet (Gitlab)  lenkes sammen med resultatet. Det går det an å teste på slik den er nå. Gender-guesser har vi (meg og Lars Magne og kanskj flere?) hatt god erfaring med.
